src/components/checkbox/index.tsx

```diff
@@ -86,8 +94,17 @@ export interface CheckboxProps extends TouchableOpacityProps {
   indeterminate?: boolean;
 }
 
+interface CheckboxMethods {
+  validate: () => void;
```

Comment:
I see in `TextField` there's a `onValidationFailed` method that **is** called when the validation fails, WDYT about adding it? Unlike `onChangeValidity` it is called if `validate` fails.
It can be another PR...

Reply:
I'll keep it simple until we'll get such a request